test/functional/ui/messages_spec.lua

```diff
@@ -1117,7 +1118,11 @@ vimComment     xxx match /\s"[^\-:.%#=*].*$/ms=s+1,lc=1  excludenl contains=@vim
   end)
 end)
 
+-- FIXME(dundar): Flaky set of tests. I suspect the before_each is faulty and that it infects all
+-- tests in the describe block.
+if global_helpers.isCI('sourcehut') then
 describe('ui/ext_messages', function()
+  pending("Flaky set of tests. The before_each function may be broken.", function() end)
```

Comment:
skip_fragile. Also don't need the isCI() condition (which looks like it should be flipped anyway?)

Reply:
>  Also don't need the isCI() condition

I think we do. `pending` doesn't work inside describe blocks.